test/components/modal.spec.js

```diff
@@ -207,6 +238,68 @@ describe('Modal spec', () => {
     })
   })
 
+  describe('bindModalShowListener', () => {
+    it('should change location hash when `modal:show` is emitted', () => {
+      const hash = '#modal-open'
+
+      emitter.emit('modal:show')
+
+      expect(window.location.hash).to.be.equal(hash)
+    })
+  })
+
+  describe('bindHashChangeListener', () => {
+    context('when there is not a location hash', () => {
+      it('should hide modal', sinon.test(function () {
+        let spy = this.spy(instance, 'hide')
```

Comment:
What do you think about using `const` over `let` in situation like this, when you don't reassign a value to the variable?